src/sandesh/Assignment_17/CountEvenOddNumbersArray.java

```diff
@@ -0,0 +1,31 @@
+package sandesh.Assignment_17;
+
+public class CountEvenOddNumbersArray {
+	static int countOfNegativeNumbers = 0;
+	int getEvenCount(int[] inArray) {
+		int evenNumbersCount = 0;
+		for(int index=0; index<inArray.length; index++) {
+			if(inArray[index] > 0) {
+				if(inArray[index] % 2 == 0)
+					evenNumbersCount++;
+			}
+			else
+				countOfNegativeNumbers++;
+		}
```

Comment:
you can also do the array.length-evenCnt to get oddCnt

Reply:
sure Harsh, will take this input.